src/main/java/org/team199/robot2020/subsystems/Intake.java

```diff
@@ -11,29 +11,34 @@
 public class Intake extends SubsystemBase {
     private final WPI_TalonSRX intakeMotor = MotorControllerFactory.createTalon(Constants.Intake.INTAKE_MOTOR);
     private final DoubleSolenoid intakePistons = new DoubleSolenoid(Constants.Intake.INTAKE_PISTONS[0], Constants.Intake.INTAKE_PISTONS[1]);
-    private final WPI_TalonSRX feederMotor = MotorControllerFactory.createTalon(Constants.Intake.FEEDER_MOTOR);
 
     public Intake(){
     }
 
     public void intake()
     {
         intakeMotor.set(Constants.Intake.INTAKE_SPEED);
-        feederMotor.set(Constants.Intake.FEEDER_SPEED);
-        intakePistons.set(DoubleSolenoid.Value.kForward); //TODO: Check if this should be kForward or kReverse
     }
 
     public void outtake()
     {
         intakeMotor.set(-Constants.Intake.INTAKE_SPEED);
-        feederMotor.set(-Constants.Intake.FEEDER_SPEED);
-        intakePistons.set(DoubleSolenoid.Value.kForward); //TODO: Check if this should be kForward or kReverse
     }
     
-    public void stopIntake()
+    public void stop()
     {
         intakeMotor.set(0);
-        feederMotor.set(0);
-        intakePistons.set(DoubleSolenoid.Value.kReverse); //TODO: Check if this should be kForward or kReverse
+    }
+
+    public void deploy() {
+        intakePistons.set(DoubleSolenoid.Value.kForward);
+    }
+
+    public void retract() {
+        intakePistons.set(DoubleSolenoid.Value.kReverse);
+    }
+
+    public boolean isRunning() {
```

Comment:
Why is this method's functionality and the way it's used the opposite of what the name suggests?